source/initial_conditions/spherical_shell.cc

```diff
@@ -157,13 +167,14 @@ namespace aspect
     SphericalGaussianPerturbation<dim>::
     initial_temperature (const Point<dim> &position) const
     {
-      // this initial condition only makes sense if the geometry is a
-      // spherical shell. verify that it is indeed
-      AssertThrow (dynamic_cast<const GeometryModel::SphericalShell<dim>*>
-                   (&this->get_geometry_model())
-                   != 0,
+      // this initial condition only makes sense if the geometry is derived from a
+      // spherical shell (i.e. a spherical shell or chunk)
+      AssertThrow ((dynamic_cast<const GeometryModel::SphericalShell<dim>*>
+                    (&this->get_geometry_model()) != 0
+                    || dynamic_cast<const GeometryModel::Chunk<dim>*>
+                    (&this->get_geometry_model()) != 0),
```

Comment:
I think you misplaced some parentheses here. Did you intend to do `if((cond1) || (cond2))`? because at the moment you do `if((x) != 0 || (y) != 0)` ... I am not sure if it wrong this way, but the inner parentheses are useless at the moment, and I would prefer the first version.